packages/wallets/src/components/WalletCard/WalletCard.scss

```diff
@@ -21,11 +21,29 @@
         align-items: flex-start;
         aspect-ratio: 1.667;
 
+        &--carousel-content {
```

Comment:
so carousel content is block, so it should be:
&__carousel-content

and then "active" is modifier, so 
&--active

Reply:
Updated